lib/manager/nuget/extract.ts

```diff
@@ -112,6 +113,42 @@ export async function extractPackageFile(
     return { deps };
   }
 
+  if (packageFile.endsWith('global.json')) {
+    const deps: PackageDependency[] = [];
```

Comment:
Please extract inner block to new function to `./extract/global-manifest.ts`

Reply:
Done.